ASTRO/ASTROpdd.tex

```diff
@@ -0,0 +1,264 @@
+
+
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+
+Deliverable items for this project include a small mid-semester presentation to outline where each of the projects are at and what needs to be done in order to complete them.  Then each team will deliver their final projects.  For each of the mount teams their respected mounts will be the delivery.  The tracking mount should also present pictures taken of celestial objects to indicate it works as desired.  The team working on the software will provide the finished code as well as a presentation outlining how to use it, how it works, as well as the technologies used.  Lastly the research team will present their findings to the group as well as create a small paper that will be reviewed by the team and, ideally, be pushed forth for publication.
+
+
+
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+Many of these projects are interdisciplinary but between them will require from ten to twelve members.  Each respected team will get anywhere between two to four members depending on the complexity and skill set of each team.  Ideally for the mounts there should be an upperclassman mechanical engineer.  For the research team there should be at least one physics related major.  Lastly for the software team it will be recommended to have one member familiar with orbital mechanics and another with general software development.
+
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+Funding requirements for these projects should be kept to a minimum.
+The mount teams will not exceed 50 dollars in materials for each respective mount and the software/research teams will require no additional funding.
+
+
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+No faculty have explicitly expressed interest in these projects. However they can be consulted if questions do arise in any stage of development.  While it is certainly feasible to do these projects without professor help it would greatly benefit the members to develop relationships with professors to help in development.
+
+\subsection{Long-Term Vision}
+\label{sec:vision}
+Once new technologies are developed the Astro Tracking team will be better prepared to conduct research in the future.  The mounts allow for obtaining better data during observations, the software will allow for the pursuit of tracking artificial satellites, and the research team will help provide a framework for how to conduct future research.
```

Comment:
All of the objectives and projects here are valid as independent parts, but how do they come together as a whole? I'd like to see some discussion on how all of these components work together to push forward a whole vision for Astro Tracking. What is that vision?

Reply:
Good point.  Will work on that.